ui/index.html

```diff
@@ -200,6 +200,15 @@
                   title="{{ title }}">{{ value }}</button>
         </script>
 
+        <!-- Possible Job Btn span -->
```

Comment:
as mentioned earlier, I'd vote for adding this to the action list instead of a button here.  :)